templates/data/postgresql/what-is-postgresql.html

```diff
@@ -0,0 +1,636 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1GPCEnZgCfYvvPXn57iKL673PfNDkXkpY9fcGTxhNefg/edit
+{% endblock meta_copydoc %}
+
+{% block title %}What is PostgreSQL?{% endblock %}
+
+{% block meta_description %}
+  PostgreSQL is the world's most acclaimed open source database management system. Explore PostgreSQL, its ecosystem, its internal architecture and how it compares to other databases.
+{% endblock %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+
+  <section class="p-strip is-shallow u-no-padding--bottom">
+    <div class="p-section--deep">
+      <div class="row--50-50-on-large">
+        <div class="col">
+          <h1>What is PostgreSQL?</h1>
+        </div>
+        <div class="col">
+          <p>
+            PostgreSQL, or just Postgres, is an open-source database management system (DBMS). It is known for its excellent support of the SQL standard, its extensibility and its enterprise-grade features &ndash; including procedural languages, extensive index types and comprehensive security.
+          </p>
+          <p>
+            PostgreSQL has been successfully used for more than three decades in all IT sectors. Its maturity and its vibrant community consistently make it a first choice database among developers.
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--25-75">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2 class="p-text--small-caps">Why choose Postgresql?</h2>
+      </div>
+      <div class="col">
+        <ul class="p-list--divided">
+          <li class="p-list__item u-no-padding">
+            <p class="p-heading--2">Comprehensive security features</p>
+          </li>
+          <li class="p-list__item u-no-padding--bottom">
+            <p class="p-heading--2">Versatile thanks to hundreds of extensions</p>
+          </li>
+          <li class="p-list__item u-no-padding--bottom">
+            <p class="p-heading--2">Reliable and proven with a 30 year track record</p>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="u-fixed-width p-section--shallow">
+      <hr />
```

Comment:
Pls either use p-rule everywhere or nowhere :)

Reply:
I have [reached out to the Vanilla team](https://chat.canonical.com/canonical/pl/9wf4my9kbt8qmdz83xn1as3b4e) about this, as currently the `<hr>` changes aren't live. I don't want to push a broken design to prod, so it depends when will we see the changes reflected in Vanilla